installer/targets/cmdgen/install.bash

```diff
@@ -8,15 +8,30 @@ then
     echo "Installing dependency of GPSRCmdGen: mono-complete"
     sudo apt-get install mono-complete
 
-    tue-install-git http://github.com/kyordhel/GPSRCmdGen.git ~/src/GPSRCmdGen
+    tue-install-git https://github.com/tue-robotics/GPSRCmdGen.git ~/src/GPSRCmdGen
 
     echo "Making GPSRCmdGen"
     cd ~/src/GPSRCmdGen
     make
 
-    tue-install-info "GPSRCmdGen is only pulled and maked once. In the future you have to do this yourself.
-               cd ~/src/GPSRCmdGen
-               make"
-else
-    tue-install-debug "GPSRCmdGen already installed"
+else # GPSRCmdGen installed
+    cd ~/src/GPSRCmdGen
+    REMOTE=$(git config --get remote.origin.url)
+    if [ "$REMOTE" != "https://github.com/tue-robotics/GPSRCmdGen.git" ]; then
+        echo "The GPSRCmdGen is still pointing to old remote, will be changed to tue-fork"
+        git remote set-url origin https://github.com/tue-robotics/GPSRCmdGen.git
+    else
+      echo "The GPSRCmdGen is correctly pointing to tue-fork"
+    fi
+    echo "Pulling GPSRCmdGen"
+    prev=$(git rev-list HEAD -n 1)
+    git pull origin master
```

Comment:
git pull origin master only updates the local master branch, not, let's say, the branch you're currently working on. Is this the behavior you want?